docs/internals/garmin-xhd-output.md

```
@@ -0,0 +1,222 @@
+### Critical constraints
+
+The Garmin plotter crashes or freezes if these fields are wrong:
+
+| Field | Offset | Required value |
+|-------|--------|----------------|
+| `fill_1` | byte 8 | `1` (not 0) |
+| `fills_4` | byte 28 | `0x0108` |
+| angle | byte 12 | 0..11512, multiple of 8 |
+
+The angle constraint is enforced by:
+```rust
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)